Core/src/org/sleuthkit/autopsy/casemodule/Case.java

```diff
@@ -583,24 +583,43 @@ public static boolean isCaseOpen() {
     }
 
     /**
+     * Deprecated. Use getOpenCase() instead.
+     * 
```

Comment:
As noted previously, this is not the correct way to document a deprecated method. Look, for example, at Case.addEventSubscriber.